src/jest/config.json

```diff
@@ -13,7 +13,8 @@
     "!ui_framework/components/**/*/index.js"
   ],
   "moduleNameMapper": {
-    "^ui_framework/components": "<rootDir>/ui_framework/components"
+    "^ui_framework/components": "<rootDir>/ui_framework/components",
+    "\\.(css|less|scss)$": "<rootDir>/src/jest/mockCssFiles.js"
```

Comment:
"Start simple" would be my reason. If this list was to exactly match the webpack loader list I would expect it to be referenced in a DRY fashion.